core/platform/cloud_translate/cloud_translate_emulator.py

```diff
@@ -0,0 +1,107 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""An emulator that mocks the core.platform.cloud_translate API. This emulator
+models the Cloud Translate API.
+"""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals # pylint: disable=import-only-modules
+
+import python_utils
+import utils
+
+
+class CloudTranslateEmulator(python_utils.OBJECT):
+    """The emulator mocks the translate_text function from the Cloud Translate
+    API. This emulator can be used in backend testing, or a local dev
+    environment without access to the Cloud Translate API. Expected responses
+    must be passed in before using this emulator for testing. See
+    PREGENERATED_TRANSLATIONS above for some prepopulated responses.
+    """
+
+    def __init__(self):
+        """Initializes the emulator with no expected responses."""
```

Comment:
Seems this is wrong, since we're assigning a non-empty value to self.expected_responses. Please fix the docstring or make the responses actually empty.

Reply:
Thanks for catching this!
Fixed.